rg_shipit/classes/ShipitSize.php

```diff
@@ -0,0 +1,68 @@
+<?php 
+  class ShipitSize {
+    public $width = 0.0;
+    public $height = 0.0;
+    public $length = 0.0;
+    public $weight = 0.0;
+    public $volumetric_weight = 0.0;
+    public $store = true;
+    public $packing_id = null;
+    public $name = '';
+
+    public function __construct($id_cart, $store = true, $packing_id = null, $name = '') {
+      $cache = new ShipitCache($id_cart);
+      $this->width = (float)$cache->package['width'];
+      $this->height = (float)$cache->package['height'];
+      $this->length = (float)$cache->package['depth'];
+      $this->weight = (float)$cache->package['weight'];
+      $this->volumetric_weight = (float)$cache->package['width']*(float)$cache->package['height']*(float)$cache->package['depth'];
```

Comment:
I see you rather prefer to take your own formula to calc volumetric weight field but, just to clarify, our couriers have their own logic to calc that field, for ex: Chilexpress is something like (width*height*depth)/4000 and others can apply the same or more.

Also I'm not aware if you are using that field at all so I will leave it as an informative warning.

* Side Note: the formula that you are applying there must give you something in orther of thousands and volumetric weight in almost any case go beyond 100